ts/screens/preferences/OldServicesHomeScreen.tsx

```diff
@@ -20,36 +22,74 @@ import Markdown from "../../components/ui/Markdown";
 import I18n from "../../i18n";
 import { contentServiceLoad } from "../../store/actions/content";
 import { navigateToOldServiceDetailsScreen } from "../../store/actions/navigation";
+import { serviceAlertDisplayedOnceSuccess } from "../../store/actions/persistedPreferences";
 import { profileUpsert } from "../../store/actions/profile";
 import {
   loadVisibleServices,
   showServiceDetails
 } from "../../store/actions/services";
 import { Dispatch, ReduxProps } from "../../store/actions/types";
-import { readServicesSelector } from "../../store/reducers/entities/services/readStateByServiceId";
-
 import { organizationNamesByFiscalCodeSelector } from "../../store/reducers/entities/organizations/organizationsByFiscalCodeReducer";
+import { readServicesSelector } from "../../store/reducers/entities/services/readStateByServiceId";
 import { ProfileState } from "../../store/reducers/profile";
 import {
   isSearchServicesEnabledSelector,
   searchTextSelector
 } from "../../store/reducers/search";
 import { GlobalState } from "../../store/reducers/types";
+import customVariables from "../../theme/variables";
 import { InferNavigationParams } from "../../types/react";
 import { isDefined } from "../../utils/guards";
-import { getChannelsforServicesList } from "./common";
+import {
+  getChannelsforServicesList,
+  getEnabledChannelsForService
+} from "./common";
 import OldServiceDetailsScreen from "./OldServiceDetailsScreen";
 
+type State = {
+  enableServices: boolean;
+  isLongPressModeEnabled: boolean;
+};
+
 type OwnProps = NavigationInjectedProps;
 
 type Props = ReturnType<typeof mapStateToProps> &
   ReturnType<typeof mapDispatchToProps> &
   ReduxProps &
   OwnProps;
 
-class OldServicesHomeScreen extends React.Component<Props> {
+const styles = StyleSheet.create({
+  listWrapper: {
+    flex: 1
+  },
+  buttonBar: {
```

Comment:
Are you sure about this style?
If the long press mode is enabled you can't reach the last item
<img src="https://user-images.githubusercontent.com/822471/62714288-91c5c580-b9fe-11e9-9bc8-2a650eb4e626.png" height="500">

Reply:
ops I hadn't noticed this problem, I have to check it

Reply:
Ok, i found the solution